analyzer/analyzer.go

```diff
@@ -102,7 +139,7 @@ func (n *perfSprint) run(pass *analysis.Pass) (interface{}, error) {
 			err   error
 		)
 		switch {
-		case calledObj == fmtErrorfObj && len(call.Args) == 1 && n.errorf:
+		case calledObj == fmtErrorfObj && len(call.Args) == 1 && n.errFormat.errorf && n.errFormat.enabled:
```

Comment:
You applied my [previous suggestion](https://github.com/catenacyber/perfsprint/pull/41#discussion_r1934065707) to change things. So you added this 0375e34eab8f3e29253ea3fafc239337bfec181b

So now all code should be updated to do not check for the ".enabled" flag

And commit 2518a2cf3035db43aa99ea29fe26c514e85b0706 could be removed from history

Reply:
Added a new commit to simplify enjoying the consistency from init